platform-sdk/swirlds-benchmarks/src/jmh/java/com/swirlds/benchmark/config/BenchmarkConfig.java

```diff
@@ -54,17 +57,16 @@ public record BenchmarkConfig(
         @ConfigProperty(defaultValue = "false") boolean csvAppend,
         @ConfigProperty(defaultValue = "sda") String deviceName) {
     public String toString() {
-        return String.format(
-                "[benchmarkData=\"%s\", saveDataDirectory=%b, verifyResult=%b, printHistogram=%b,"
-                        + " csvOutputFolder=\"%s\", csvFileName=\"%s\", csvWriteFrequency=%d, csvAppend=%b, deviceName=\"%s\"]",
-                benchmarkData,
-                saveDataDirectory,
-                verifyResult,
-                printHistogram,
-                csvOutputFolder,
-                csvFileName,
-                csvWriteFrequency,
-                csvAppend,
-                deviceName);
+        return new ToStringBuilder(this, SHORT_PREFIX_STYLE)
```

Comment:
ToStringBuilder isn't really faster than String.format (and not any clearer than STRING_CONSTANT.formatted(arguments...)), in fact it's arguably slower.  I'm not sure replacing format strings with a builder from apache lang3 is a great approach.  I can definitely see replacing the concatenations elsewhere with a better approach, but this case is less clearly beneficial.

Reply:
The idea was not to make it faster but rather make it clear-looking and more readable.

Reply:
In fact, I'm open to an idea of using an approach that is more performant but I think it has to be used consistently across all the code base. I'd like to standardize it first by migrating to `ToStringBuilder` and then, if the need arises, we can migrate it all together to something else.

Reply:
I find String.formatted to be far clearer and easier to read than ToStringBuilder and its analogs, but I can see value in making that change as a second phase.  I worry, a bit, that we won't ever make that change and this model will become a preferred style, effectively locking us into the lesser design.